src/app/modules/programs/components/edit-program/edit-program.component.ts

```diff
@@ -123,13 +116,25 @@ export class EditProgramsComponent implements OnInit {
     private readonly modalService: NgbModal,
     private readonly toastService: ToastService,
     private readonly replaceInTranslationPipe: ReplaceInTranslationPipe,
+    private readonly router: Router,
   ) {}
 
   ngOnInit(): void {
     const data = this.resolversService.getDataFromPathFromRoot(this.activatedRoute.pathFromRoot);
     const tags = (data[EResolvers.LeadResolver] as ILeadData).tags;
     this.company = (data[EResolvers.AppResolver] as IAppData).company;
-    this.program = (data[EResolvers.EditProgramResolver] as IEditProgramData).program;
+    const editProgramData = data[EResolvers.EditProgramResolver] as IEditProgramData;
+    this.program = editProgramData.program;
+
+    this.tabsControl.setValue(
+      this.tabsOptions.find((tab) => tab.value === editProgramData.tab) ?? this.tabsOptions[0],
+    );
+
+    this.editProgramComponentSubscription.add(
```

Comment:
We just need to update url without navigating (see lead-collaboration.component). Navigating as such will trigger the resolver and we don't need this behavior.